README.md

```diff
@@ -1,3 +1,96 @@
-saml-ios-app
-============
-Simple native iOS app to work with [```SAML Service``` connector service](https://github.com/feedhenry-templates/saml-service). To configure the service in your RHMAP platform read the [SAML notes](https://github.com/feedhenry-templates/saml-service/blob/master/NOTES.md)
\ No newline at end of file
+# SAML iOS Client
+
+## What does the demo do?
+
+This is an example SAML Client App, designed to be used in conjunction with our [SAML Service](https://github.com/feedhenry-templates/saml-service). 
+
+The Cloud App proxies the login client call to a SAML Service (e.g. fetching a URL to display in the WebView for IdP login) through the use of a SAML service (See Configuration section).
+
+The client app demoes the usage of the two SAML Service endpoints:
+- `sso/session/login_host` to get the SAML login page URL. A WebView is used to display the SAML login page.
+- `sso/session/valid` to get the user information once the login was successful. 
+
+Before running the project you will need some configuration.
+
+## Configuration
```

Comment:
@corinnekrych @edewit This section is note about configuration for client app, IMHO it need to be moved to the service, cloud or client template. Wdyt?